### PR TITLE
NXP-28744: upgrade Apache POI and adapt JXLS 1

### PR DESCRIPTION
**Pull request by efge**
https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-fguillaume/780/

### CODE REVIEW

File: nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/advanced/AdvancedMSOfficeConverterTest.java [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
 public class AdvancedMSOfficeConverterTest extends SimpleConverterTest {
 
+    protected String expectedContentFilename;
```
Comment (ataillefer) [inline nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/advanced/AdvancedMSOfficeConverterTest.java]: Seems never set, how can this work :thinking:

Comment (efge) [reply nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/advanced/AdvancedMSOfficeConverterTest.java]: It's set in the subclass — I didn't want to change the whole API of `SimpleConverterTest`

**Comment by efge** [comment]
FYI the JXLS changes in 1.0.6-NX02 are https://github.com/nuxeo/jxls-1/commit/3a621ed59c5cbbf11eb5b6f8493ea1a3ba4e52ad, see https://jira.nuxeo.com/browse/VEND-46

File: nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/advanced/AdvancedMSOfficeConverterTest.java (hunk 1 above)
Comment (ataillefer) [reply nuxeo-core/nuxeo-core-convert-plugins-test/src/test/java/org/nuxeo/ecm/core/convert/plugins/tests/advanced/AdvancedMSOfficeConverterTest.java]: OK, still sounds weird but fine.
Anyway, there is only one subclass of `AdvancedMSOfficeConverterTest`...

**Pull request merged**